child-project-2/src/main/java/com/example/features/MySampleFileSystemProviderFeature.java

```diff
@@ -0,0 +1,15 @@
+package com.example.features;
+
+import com.oracle.svm.core.annotate.AutomaticFeature;
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeClassInitialization;
+
+@AutomaticFeature
+final class MySampleFileSystemProviderFeature implements Feature {
+  private static final String clazz = "com.example.MySampleFileSystemProvider";
+
+  @Override
+  public void beforeAnalysis(BeforeAnalysisAccess access) {
+    RuntimeClassInitialization.initializeAtBuildTime(clazz);
```

Comment:
From the document https://docs.oracle.com/en/graalvm/enterprise/22/docs/reference-manual/native-image/ClassInitialization/